release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1557,7 +1555,34 @@ For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=19300
+* Previously, an incorrect code was causing the `User` and `Group` details to show unrelated subjects. Now, a code has been added to filter by `User` or `Group`, so the `User` and `Group` details show related subjects. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1951212[*BZ#1951212*])
+
+* Previously, when you imported a devfile, the OpenShift web console ignored the `build guidance` placeholder container which provided the configuration for environment variables, ports and limits. The new deployment had a second container which could not start because the placeholder image could not be fetched and it missed the required configurations. Now, the `build guidance` container is dropped from the new deployment, and the container adds the environment variables, ports and limit configurations. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1952214[BZ#1952214])
+
+* Previously, routes tied to the *Developer* perspective were not rendered. This produced 404 error messages when you refreshed the *Developer* perspective. The current release fixes this issue by loading all the inactive routes. Now, the routes switch to the correct perspective before loading. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1929769[BZ#1929769])
+
+* Previously, the colors for the *Low* and *Medium* severity issues of the Image Manifest Vulnerabilities (IMVs) did not match the color representation shown in the (link:https://quay.io/[Quay.io]) interface. As a result, when the user changed the severity order of vulnerabilities to *High*, the IMVs ordered the issues incorrectly. This created confusion when reviewing the IMVs. The current release fixes this issue. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1942716[BZ#1942716])
+
+* Previously, if the API server encountered a conflict while updating a resource quota, it sometimes failed to create a resource and returned a 409 status code. In that case, the client had to retry the API request. With this update, when the API server encounters this type of conflict, it retries creating the resource up to three times. This is often sufficient to create the resource. Otherwise, the API server displays an error in the console. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1920699[BZ#1920699])
+
+* Previously, pipelines created in the Git import flow for private repositories failed to run. This happened because the pipeline `ServiceAccount` did not use secrets created by the Git import flow for private Git repositories. With this update, you can add a secret name to the annotations of the pipeline `ServiceAccount` object, and add pipeline-specific annotations to the provided secret. As a result, pipelines for private Git repositories run successfully. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1970470[BZ#1970470])
```

Comment:
Please change the second sentence to - This happened because the pipeline `ServiceAccount` object did not use...

Reply:
This applies to PR 34416 too: https://github.com/openshift/openshift-docs/pull/34416/files#diff-14b6355bc5db567351b9e43a469c73396794e644c2dd1fc8523417f2f7b72c16R1303

Reply:
done